benchmarks/solcx/compositional_fields/solcx_particles.prm

```diff
@@ -0,0 +1,133 @@
+# A description of the SolCX benchmark using active particles
```

Comment:
But did we not create the new directory `compositional_fields` for exactly the reason to separate the compositional field case from the particle case? And the particle case in the `particle` subfolder would get its own shared library that also just needs to include the files from `benchmarks/solcx/`? At least that is how I remember this, did I mix that up in my mind?